operators/vision/encode_image.hpp

```diff
@@ -18,12 +30,58 @@ struct KernelEncodeImage : BaseKernel {
     }
 
     extension_ = std::string(".") + format;
+    encoder_.OnInit();
   }
 
-  void Compute(const ortc::Tensor<uint8_t>& input_bgr,
-               ortc::Tensor<uint8_t>& output) const;
+  void Compute(const ortc::Tensor<uint8_t>& input_bgr, ortc::Tensor<uint8_t>& output) const{
```

Comment:
Yes. but we can add an attribute for this class to support both.

Reply:
I think an attribute is much needed for clarification. Because from the shape, we can't reliably determine if a bunch of bytes is RGB or BGR. They look exactly the same from the callers' point of view..